README.md

```diff
@@ -86,6 +86,21 @@ to install the default SSSP version.
 List the installed pseudopotential families with the command `aiida-pseudo list`.
 You can then use the name of any family in the command line using the `-F` flag.
 
+## Development
+
+### Test
+```shell
+git clone https://github.com/aiidateam/aiida-quantumespresso .
+cd aiida-quantumespresso
+pip install -e .[tests]  # install extra dependencies for test
+pytest tests/calculations/test_pw.py # run tests
```

Comment:
Maybe we want to show how to run all tests by default?

```suggestion
pytest # run tests
```

You can also add an example to show how to run a specific set of tests, but we should assume a developer understands the basics of `pytest`.